mail/dovecot/Portfile

```diff
@@ -1,110 +1,453 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem          1.0
+PortGroup           github 1.0
 
+github.setup        dovecot core 2.3.9.2
 name                dovecot
+revision            0
 conflicts           dovecot2
 epoch               20060722
-version             1.2.17
-revision            2
-set branch          [join [lrange [split ${version} .] 0 1] .]
+set core_version    ${version}
+
 categories          mail
-license             {MIT LGPL-2.1 BSD public-domain}
-maintainers         {jberry @jdberry} openmaintainer
+maintainers         maintainers {jberry @jdberry} openmaintainer
```

Comment:
This has an extra "maintainers" in it that needs to be removed.

Reply:
Oooops. Thanks. Fixed.